python/activator/activator.py

```diff
@@ -32,12 +32,14 @@
 from flask import Flask, request
 from google.cloud import pubsub_v1, storage
 
+from lsst.daf.butler import Butler
 from .middleware_interface import MiddlewareInterface
 from .visit import Visit
 
 PROJECT_ID = "prompt-proto"
 
 verification_token = os.environ["PUBSUB_VERIFICATION_TOKEN"]
+# The full instrument class name, including module path.
```

Comment:
I realize it's not going to be runnable as-is, but can you update the [Playbook](https://github.com/lsst-dm/prompt_prototype/blob/main/doc/playbook.rst#prototype-service) accordingly?